lib/migration.js

```diff
@@ -80,39 +68,39 @@ module.exports = function(DB2iSeries) {
     });
   };
 
-  DB2iSeries.prototype.showFields = function(model, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{showFields()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.showFields = function(model, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{showFields()}} is not currently supported.')));
+//     });
+//   };
 
-  DB2iSeries.prototype.showIndexes = function(model, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{showIndexes()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.showIndexes = function(model, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{showIndexes()}} is not currently supported.')));
+//     });
+//   };
 
-  DB2iSeries.prototype.autoupdate = function(models, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{autoupdate()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.autoupdate = function(models, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{autoupdate()}} is not currently supported.')));
+//     });
+//   };
 
   DB2iSeries.prototype.isActual = function(models, cb) {
     process.nextTick(function() {
       return cb(Error(g.f('{{isActual()}} is not currently supported.')));
     });
   };
 
-  DB2iSeries.prototype.alterTable = function(model, fields, indexes, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{alterTable()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.alterTable = function(model, fields, indexes, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{alterTable()}} is not currently supported.')));
+//     });
+//   };
 
-  DB2iSeries.prototype.getTableStatus = function(model, cb) {
-    process.nextTick(function() {
-      return cb(Error(g.f('{{getTableStatus()}} is not currently supported.')));
-    });
-  };
+//   DB2iSeries.prototype.getTableStatus = function(model, cb) {
+//     process.nextTick(function() {
+//       return cb(Error(g.f('{{getTableStatus()}} is not currently supported.')));
+//     });
+//   };
```

Comment:
Had a quick look.  I'm going to state that migration is not supported at this time for this connector as it is going to require writing these functions from scratch as the system catalogs are different than those from the other IBM databases.  We'll have to do a minor update to add migration/discovery later.